project/plugins.sbt

```diff
@@ -13,5 +13,6 @@ addSbtPlugin("org.scalastyle" %% "scalastyle-sbt-plugin" % "1.0.0")
 addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.14.6")
 addSbtPlugin("com.simplytyped" % "sbt-antlr4" % "0.8.1")
 addSbtPlugin("com.github.gseitz" % "sbt-protobuf" % "0.6.3")
+addSbtPlugin("ch.epfl.scala" % "sbt-scalafix" % "0.9.4")
```

Comment:
Similar to John's question... why is this needed?

Reply:
FIRRTL uses this plugin

https://github.com/freechipsproject/firrtl/blob/master/project/plugins.sbt#L23